### ISSUE TITLE
[compute] Porting Q4_0 params Gather compute library

### ISSUE THREAD

**Issue by hseok-oh** [Samsung/ONE#13910]
### What

Porting ggml compuation library: Q4_0 Gather (get_rows) into `compute/cker` or better place, and called from kernel generator

### Why

Support block quantized Gather layer in CPU backend

**Comment by hseok-oh** [comment]
`Gather` operation restriction to use ggml's `get_rows` computation 

- `input` (`params` in tensorflow )
  - Rank: 2
- `indices` (`index` in pytorch)
  - Type: INT32
  - Rank: 3 or 4
    - Shape is [1, a, b, c] if rank is 4
- `axis` (`dim` in pytorch)
  -  Value: 0

**Comment by hseok-oh** [comment]
Draft: #13956

**Comment by hseok-oh** [comment]
Done
